app/components/emoji_picker/emoji_picker_base.js

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+/* eslint-disable max-lines */
```

Comment:
lol ok.. the reality is that this entire file should be refactored, but out of scope for this PR.

Reply:
yeah, but IDE marks everything in yellow because of that and makes it harder to see an error/warning. I don't like doing that, but refactoring the file would be too much out of scope :D